common/python/schema_validation/data/WorkOrderSubmit.json

```diff
@@ -0,0 +1,169 @@
+{
+"type": "object",
+"uniqueItems": true,
+"additionalProperties": false,
+"properties": {
+    "responseTimeoutMSecs": {
+        "type": "integer",
+        "error_msg":
+        "Invalid data format for responseTimeoutMSecs"},
+    "payloadFormat": {
+        "type": "string",
+        "enum": ["JSON-RPC", "json-rpc"],
+        "error_msg":
+        "Invalid payload format"},
+    "resultUri": {
+        "type": "string",
+        "error_msg":
+        "Invalid data format for resultUri"},
+    "notifyUri": {
+        "type": "string",
+        "error_msg":
+        "Invalid data format for notifyUri"},
+    "workOrderId": {
+        "type": "string",
+        "pattern": "^(0[x|X])?[0-9a-fA-F]+$",
+        "error_msg":
+        "Invalid data format for work order id"},
+    "workerId": {
+        "type": "string",
+        "pattern": "^(0[x|X])?[0-9a-fA-F]+$",
+        "error_msg":
+        "Invalid data format for Worker id"},
+    "workloadId": {
+        "type": "string",
+        "pattern": "^(0[x|X])?[0-9a-fA-F]+$",
+        "error_msg":
+        "Invalid data format for work load id"},
+    "requesterId": {
+        "type": "string",
+        "pattern": "^(0[x|X])?[0-9a-fA-F]+$",
+        "error_msg":
+        "Invalid data format for requester id"},
+    "workerEncryptionKey": {
+        "type": "string",
```

Comment:
Let us state the expectation for now in some GitHub issue. Only hex will be accepted.
Later, we need to revisit to see what flow can b taken for the lack of a regex for base64 encoded key.